receiver/prometheusreceiver/metrics_receiver_helper_test.go

```diff
@@ -474,14 +479,17 @@ func testComponent(t *testing.T, targets []*testData, useStartTimeMetric bool, s
 	// loop to validate outputs for each targets
 	for _, target := range targets {
 		t.Run(target.name, func(t *testing.T) {
-			validScrapes := getValidScrapes(t, pResults[target.name])
+			validScrapes := pResults[target.name]
+			if !useOpenMetrics {
```

Comment:
Does OpenMetrics not include failed scrapes or scrapes with only scrape metrics?  Why the different handling here?

Reply:
`getValidScrapes` asserts that the _up_ metric value is 0 whenever the scrape only has the default scrape metrics and no other metric ([Line 204](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/eacfde3fcbd46ba60a6db0e9a41977390c4883bd/receiver/prometheusreceiver/metrics_receiver_helper_test.go#L204))
Some of the negative tests generate only the default scrape metrics with up value 1 due to which the test would report failure inside getValidScrapes before they are skipped in `verifyNegativeTarget`.

Reply:
Makes sense.  Thanks.